### DISCUSSION TITLE
OhMyPosh not visible?

### DISCUSSION THREAD

**Discussion by dominikjeske** [JanDeDobbeleer/oh-my-posh#4919] [Troubleshoot]
Recently I updated all my tools

- OhMyPosh 19.27.0
- Powershell 7.4.2
- WindowsTerminal 1.21.1272

After this when I start new command my terminal is empty (I see only blinking cursor) and I have to write something or resize window and then everything back to normal. But when I invoke new command problem starts again. Any ideas?

**Comment by JanDeDobbeleer** [comment]
This seems to be an issue in either the latest pwsh or terminal as this didn't happen before and nothing changed in that area in a while.
